syntax = "proto3";

package FoodBot;

service FoodBotRequest
{
//Input NLG
//Output NLG 
	rpc GetResponse(Sentence) returns (outSentence){}
}

message Sentence
{
	string semantic_frame = 1;
	string nlg_sentence = 2;
	string user_id = 3;
}

message outSentence
{
	string response_nlg = 4;
	string response_policy_frame = 5;
	string url = 6;
}